src/Microsoft.VisualStudio.Threading.Analyzers/Microsoft.VisualStudio.Threading.Analyzers.csproj

```diff
@@ -34,105 +28,50 @@
-    <EmbeddedResource Include="Strings.cs.resx" />
-    <EmbeddedResource Include="Strings.de.resx" />
-    <EmbeddedResource Include="Strings.es.resx" />
-    <EmbeddedResource Include="Strings.fr.resx" />
-    <EmbeddedResource Include="Strings.it.resx" />
-    <EmbeddedResource Include="Strings.ja.resx" />
-    <EmbeddedResource Include="Strings.ko.resx" />
-    <EmbeddedResource Include="Strings.pl.resx" />
-    <EmbeddedResource Include="Strings.pt-BR.resx" />
-    <EmbeddedResource Include="Strings.resx">
+    <EmbeddedResource Update="Strings.resx">
       <Generator>ResXFileCodeGenerator</Generator>
       <LastGenOutput>Strings.Designer.cs</LastGenOutput>
     </EmbeddedResource>
-    <EmbeddedResource Include="Strings.ru.resx" />
-    <EmbeddedResource Include="Strings.tr.resx" />
-    <EmbeddedResource Include="Strings.zh-Hans.resx" />
-    <EmbeddedResource Include="Strings.zh-Hant.resx" />
   </ItemGroup>
   <ItemGroup>
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.cs.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.de.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.es.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.fr.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.it.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.ja.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.ko.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.pl.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.pt-BR.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.ru.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.tr.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.zh-Hans.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.zh-Hant.xlf" />
+    <Content Include="tools\*.ps1" Pack="true" PackagePath="tools\" />
+  </ItemGroup>
+  <ItemGroup>
+     <XliffResource Include="MultilingualResources\*.xlf" />
   </ItemGroup>
   <ItemDefinitionGroup>
-    <Reference>
-      <!-- Prevent all references from being packaged since VS supplies them. -->
-      <ExcludeFromNuPkg>true</ExcludeFromNuPkg>
-    </Reference>
+    <PackageReference>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
   </ItemDefinitionGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="1.2.0" />
```

Comment:
What made you decide what to keep and what to update for the package references? I see these are some of the packages in project.json but some are either missing or updated to a newer version